doc/specs/spec-template.md

```diff
@@ -0,0 +1,50 @@
+---
+author: <first-name> <last-name> <github-id>/<email>
+created on: <yyyy-mm-dd>
+last updated: <yyyy-mm-dd>
+issue id: <github issue id>
+---
+
+# Spec Title
+
+## Abstract
+
+[comment]: # Outline what this spec describes

```

Comment:
Do you want to be comments like these, or HTML comments, like in the issue/PR template?

Reply:
```[comment]: <> (comment-text)``` appears to be [the preferred, most portable way of writing source-side comments](https://stackoverflow.com/a/20885980) in markdown